src/Game/Gui/Inventory/InventoryMidPane.cs

```diff
@@ -19,10 +19,7 @@ protected override void Subscribed()
             var positions = config.Inventory.Positions[_activeCharacter];
             var backgroundStack = new FixedPositionStack();
             var background = new UiSpriteElement<FullBodyPictureId>((FullBodyPictureId)_activeCharacter);
-            var backgroundButton = new Button(background) { Theme = ButtonTheme.Invisible }
-                .OnClick(() => Raise(new InventorySwapEvent(InventoryType.Player, (ushort)_activeCharacter, ItemSlotId.CharacterBody)));
-            backgroundStack.Add(backgroundButton, 1, -3);
-            AttachChild(backgroundStack);
+            backgroundStack.Add(background, 3, 10 - 1);
```

Comment:
Why 10 - 1 rather than just 9?

Reply:
Because (3, 10) is the offset of the frame, but the character sprites start one pixel above the frame, so at (3, 9). So they're 1px above the frame, which is one of the reasons why they can't be contained inside the frame. You can see this with Drirr's and Sira's image. Should I make that clearer in a comment or just write 9?

Reply:
I'm fine with keeping it as 10 - 1, I'd just put a little comment on the end like "// -1 to adjust for frame" or something.